src/test/java/net/snowflake/ingest/streaming/internal/FlushServiceTest.java

```diff
@@ -48,11 +48,22 @@
 import net.snowflake.ingest.utils.SFException;
 import org.junit.Assert;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
 import org.mockito.ArgumentCaptor;
 import org.mockito.ArgumentMatchers;
 import org.mockito.Mockito;
 
+@RunWith(Parameterized.class)
 public class FlushServiceTest {
+
+  @Parameterized.Parameters(name = "isIcebergMode: {0}")
```

Comment:
I thought what we should be doing is to test the isIcebergMode with {true, false}

Reply:
This line is the format of the test name, where the first parameter is either `false` or `true` from [here](https://github.com/snowflakedb/snowflake-ingest-java/blob/6c9bc9b3e3b9bd687898aeb59cb0d30940b4c8c9/src/test/java/net/snowflake/ingest/streaming/internal/FlushServiceTest.java#L62).